app/src/main/res/values/strings.xml

```diff
@@ -571,39 +571,37 @@
 
-        <h5>• I tried to login via Access Token &amp; OTP but it does not work?</h5>
-        <p>You can\'t login via Access Token &amp; OTP all together due to the lifetime of the OTP code, you\'ll be required to login in every few seconds.</p>
+        <h5>• I have tried signing in with an Access Token and a One-Time Password (OTP) without any luck. What's wrong?</h5>
+        <p>You can\'t sign in using Access Token and OTP all together. Due to the limited lifetime of such passwords you\'ll have to repeat a sign in procedure every few seconds.</p>
 
-        <h5>• Why are my Private Repo &amp; Enterprise Wiki not showing up?</h5>
-        <p>It\'s due to FastHub scraping GitHub Wiki page &amp; Private Repos require session token that FastHub is unable to obtain.</p>
+        <h5>• Why isn\'t my Private/Enterprise Repository\'s Wiki showing up?</h5>
+        <p>As FastHub scraps GitHub's Wiki pages, the application is unable to obtain any session token which is required by Private Repositories.</p>
 
-        <h5>• I login with Enterprise account but can\'t interact with anything other than my Enterprise GitHub?</h5>
-        <p>Well, logically, you can\'t access anything else other than your Enterprise.  FastHub tries to allow as much possible, but can\'t do much about it
-        in most cases, since your login credential doesn\'t exists in GitHub server. But in a <b>few</b>
-        cases your GitHub account Oauth token will do the trick.</p>
+        <h5>• I have signed into my Enterprise account. Why can\'t I interact with anything other than my Enterprise GitHub?</h5>
+        <p>Well, logically, you can\'t access anything else other than your Enterprise. FastHub tries to require as much permissions as possible, but can\'t do anything more about it
+        in most cases, since your credentials are not on GitHub server. But in <b>some</b> cases your GitHub account OAuth token will do the trick.</p>
 
-        <h5>• Why am I having problems editing Issues/PRs?</h5>
-        <p>If you are editing a public Org repo, then please contact your Org to grant access to FastHub or use Access Token to login!.</p>
+        <h5>• Why am I having problems editing Issues/Pull Requests (PRs)?</h5>
+        <p>If you are unable to edit Issues/PRs in a Public Organization's repository, please contact your Organization to grant access to FastHub. Alternatively you can sign in with an Access Token.</p>
 
-        <h5>• I\'m having this issue or I want this &amp; that!!</h5>
-        <p>Head to https://github.com/k0shk0sh/FastHub/issues/new and create new issue for bugs or feature requests, I really do encourage you to
-        search before opening a ticket. Any duplicate request will result in it being closed immediately.</p>
+        <h5>• I\'m having problems! / I want to suggest you some things!</h5>
+        <p>Head to https://github.com/k0shk0sh/FastHub/issues/new and create a new issue for any bugs and feature requests. I really do encourage you to
```

Comment:
Take out the word `do` before the word `encourage`

Reply:
`do` adds an emphasis to the statement.

Reply:
doesn't mean its correct english

Reply:
But it is so.

Reply:
Not that i know of

Reply:
Here's a good explanation:
https://www.italki.com/question/319207?hl=en

Reply:
Yes that makes sense if your responding to someone. (look at the example) but normally its not used..

Whatever...